src/components/analytics.js

```diff
@@ -0,0 +1,59 @@
+const client = require('prom-client')
+const http = require('http')
+
+client.collectDefaultMetrics({ timeout: 5000 })
+
+const server = http.createServer((req, res) => {
+  const reqUrl = new URL(req.url, 'http://127.0.0.1/')
```

Comment:
the connection ip might be a good thing to add to the env

Reply:
`'http://127.0.0.1/'` is the base for the URL, it ultimately does not matter since we don't use it. It's just required by the constructor since `req.url` is a relative path